README.md

```diff
@@ -1,2 +1,79 @@
-# Hello-Microverse-Review
-Setup of Microverse hello project for TSE training
+![](https://img.shields.io/badge/Hello-Microverse-blue)
+
+# Hello Microverse Project
+
+> Hello Microverse Project is simple app built  during microverse bootcamp which helps one to get familiar with setting up basic project on github
+
+![screenshot]()
+
+<br/>
+
+# Live Demo
+
+[Live Link]()
+
+<br/>
+
+## Built With
+
+
+- Technologies/tools used 
+  
+  ``` bash
+ 
+  - Git(version control)
+  - webhint(html linting)
+  - Stylelint(style linting)
+
+  ```
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+ - A text editor(preferably Visual Studio Code)
+### Install
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
+  -  [VS Code](https://code.visualstudio.com/download)
+
+#### Clone this repository
+
+```bash
+$ git clone https://github.com/mugishasam123/Hello-Microverse-Review
+.git
+$ cd Hello-Microverse-Review
+
+```
+#### Run project
+
+```bash
+
+$ open index.html in the browser 
+```
+
+  <br>
+
+## Authors
+
+
+
```

Comment:
- [ ] I would suggest you add the author of the project in your README file.  You can do that by adding all your social media links to the author section so that the people might follow more of you activities in your career.  You can use this [template ](https://github.com/microverseinc/readme-template)as your reference.